mappings/net/minecraft/class_6148.mapping

```diff
@@ -1,7 +0,0 @@
-CLASS net/minecraft/class_6148
```

Comment:
I think this should just be moved into the existing unused package next to where all of the package info's live.

Reply:
Should the class keep its mappings as well?

Reply:
Unless there is definitive proof that these are and only are constants for redstone values (also two of the fields have same value, dunno how you can tell them apart), they should not keep the mappings and the class name you give it.